splice_machine/src/main/java/com/splicemachine/derby/stream/function/csv/AbstractFileFunction.java

```diff
@@ -268,7 +270,7 @@ else if (values.size() < returnRow.nColumns()) {
         }
     }
 
-    void checkPreference() {
+    public void checkPreference() {
```

Comment:
I think this could be protected since it's only accessed from a subclass

Reply:
ok

Reply:
done